insight/tasks/user_weekly_trend_analysis.py

```
@@ -0,0 +1,160 @@
+import logging
+import asyncio
+from collections import defaultdict
+
+import aiohttp
+import setup_django  # noqa
+from django.conf import settings
+from django.db.models import Sum
```

Ruff (0.11.9)

`django.db.models.Sum` imported but unused

Remove unused import: `django.db.models.Sum`

(F401)